app/templates/base.html

```diff
@@ -121,7 +121,8 @@
                  Ministerstva zdravotnictví ČR</a> a od Centrálního řídícího týmu COVID-19 (CŘT).
             </p>
             <p class="mb-1">
-                Vydavatel: ČVUT, Fakulta informačních technologií, Praha ve spolupráci s firmou Profinit EU, s.r.o., Praha, 2021.
+                Vydavatel: ČVUT, Fakulta informačních technologií, Praha ve spolupráci s firmou Profinit EU, s.r.o., Praha, 2021.<br />
+                Citace: Sušický M., Staněk J. <i>COVID-19 očkování</i> [online]. Praha: ČVUT, Fakulta informačních technologií, 2021 [cit. {{ now | format_date_en }}]. Dostupné z https://ockovani.opendatalab.cz ISSN 2787-9925
```

Comment:
according to the citace.com, ISSN should be before URL

`...[cit. 2021-11-15]. ISSN 2787-9925. Dostupné z: https://ockovani.opendatalab.cz`

Reply:
Done